ingest/subsample.py

```diff
@@ -17,7 +17,7 @@
 class SubSample(Annotations):
     ALLOWED_FILE_TYPES = ["text/csv", "text/plain", "text/tab-separated-values"]
     MAX_THRESHOLD = 100_000
-    SUBSAMPLE_THRESHOLDS = [MAX_THRESHOLD, 20_000, 10_000, 1_000]
```

Comment:
Can we defer this removal for now?

IIRC, today's demo established that the benefits of filterable dot plots may well outweigh the costs of lossy filtered dot plot representation -- assuming we have a filter query summary and display a subsampling cavaet.

These lower subsamplings seem necessary for filterable dot plots.  One could imagine applying a 20K subsampling to a 100K clustering _only_ in the context of dot plots, and having that enable filterable dot plots for (say) 10-20 genes.  We know [some researchers find SCP dot plots very useful](https://drive.google.com/file/d/1UMwWlmlduvUwHSIuZsGacpknuOnlchxj/view?t=28m25s), and I suspect being able to filter them by annotations would boost their engagement and value quite a bit.

There are probably other ways we could scale cell filtering to enable using it in dot plots, but subsampling seems like the easiest reasonable way.  _If_ we learn that subsampling dot plots in the context of filtering -- even with cavaets displayed -- is too inaccurate or doesn't actually enable dot plot filtering, _then_ I would support removing these lower thresholds.

Reply:
Ok, sure - we can defer for now.